test/v3IOC/st.cmd

```diff
@@ -2,10 +2,6 @@
 epicsEnvSet("EPICS_CA_MAX_ARRAY_BYTES","450000")
 
 ## Load record instances
-## dbLoadRecords "db/masarExamplePart1.db"
-## dbLoadRecords "db/masarExamplePart2.db"
-## dbLoadRecords "db/masarExamplePart3.db"
-## dbLoadRecords "db/largeWf.db"
```

Comment:
I see that the files are still present.  Are they used elsewhere?  If not, they should be removed.